src/test/java/org/apache/commons/lang3/StringUtilsContainsTest.java

```diff
@@ -292,6 +292,28 @@ public void testContainsIgnoreCase_StringString() {
         assertTrue(StringUtils.containsIgnoreCase("xabcz", "ABC"));
     }
 
+    @Test
+    public void testContainsAnyIgnoreCase_StringStringArray() {
+        assertFalse(StringUtils.containsAnyIgnoreCase(null, (String[]) null));
+        assertFalse(StringUtils.containsAnyIgnoreCase(null, new String[0]));
+        assertFalse(StringUtils.containsAnyIgnoreCase(null, new String[] { "hello" }));
```

Comment:
these would precisely short-circuit at the same condition as with the assertion before these and hence might just be redundant

Reply:
Okay, I'll remove them.